pkg/manager/member/tiflash_member_manager_test.go

```diff
@@ -1186,6 +1186,7 @@ func TestGetNewServiceForTidbCluster(t *testing.T) {
 						"app.kubernetes.io/managed-by": "tidb-operator",
 						"app.kubernetes.io/instance":   "foo",
 						"app.kubernetes.io/component":  "tiflash",
+						"app.kubernetes.io/used-by":    "peer",
```

Comment:
Please also help update the labels for Pump, TiFlash and TiCDC.

Reply:
Pump, TiFlash and TiCDC also have multiply service, this label will only by when a component have mulitply service

Reply:
ok, got it.